### ISSUE TITLE
Docker support

### ISSUE THREAD

**Issue by ValdasK** [svenstaro/genact#9]
Could you create a docker image and push to docker hub for running it without downloading any additional files? This seems to be quick option to get new users as they only have to run single command instead of trying to download correct binaries and then figure out how to launch them.

**Comment by svenstaro** [comment]
I think it might be a better fix to expand the README to tell people which binaries to get and how to run them? I think using Docker for this is really the wrong way around as the project is purposefully easy to run and build.

**Comment by ValdasK** [comment]
For me Docker would be nice because I feel much safer running someone else compiled binaries in protected environment, not directly on my own machine OS.

**Comment by j796160836** [comment]
I wrote the `Dockerfile` for you as pull request #15.
Kindly check, thanks.

**Comment by svenstaro** [comment]
I merged the PR by @gaga5lala. Does that one work for you, @j796160836?

**Comment by ValdasK** [comment]
Thanks for building image. In order for other people to use it without building it locally, you should push that image to docker hub (https://hub.docker.com/)

When running without local build:

```
docker run -it --rm svenstaro/genact
Unable to find image 'svenstaro/genact:latest' locally
docker: Error response from daemon: pull access denied for svenstaro/genact, repository does not exist or may require 'docker login'.
```

**Comment by svenstaro** [comment]
Ok done, please check.

**Comment by ValdasK** [comment]
It seems to work, but I am unable to kill process using `ctrl+c`. Are those handlers set up?

**Comment by svenstaro** [comment]
Could you check out the new image? If you launch with `-it` it should work with just ctrl-c.

**Comment by ValdasK** [comment]
Something is still not right.

```
▶ docker pull svenstaro/genact        
Using default tag: latest
latest: Pulling from svenstaro/genact
Digest: sha256:7c8059986097d464b88bbbacf94f2f2e7d87d0feb599a34d343a2a91f06561ff
Status: Image is up to date for svenstaro/genact:latest

▶ docker run -it --rm svenstaro/genact
e820: remove [mem 0x000a0000-0x000fffff] usable
BRK [0x44a212000, 0x44a212fff] PGTABLE
reserve setup_data: [mem 0x0000000000000^C000-0x0000000000057fff] usable
^C^C^CNMI watchdog: Enabled. Permane^Cntly ^Cconsume^Cs one h^Cw-PMU^C count^Cer.
^C^C^C^C^C^C^C^CENER^CGY_P^CERF_BI^CAS: Vie^Cw and^C updat^Ce with^C x86_^Cenergy^C_perf_po^Clicy^C(8)
.. and it continues
```

**Comment by j796160836** [comment]
The edited version of @svenstaro  works of me, thanks.

@ValdasK Use `-t` instead of `-it`. 
```
docker run -t --rm svenstaro/genact
```

But there comes up a problem that container won't stop with `Ctrl+C`.
Maybe a bug?

**Comment by yunake** [comment]
yep facing the same issue with SIGINT on Arch

**Comment by svenstaro** [comment]
Not quite sure what to do here, to be quite honest. Can anyone with experience on this weigh in?

**Comment by gaga5lala** [comment]
Try to execute genact in docker container, and ctrl-c work fine.
But I have no idea why it broke when using docker run.

```
$ docker run -it --rm --name genact svenstaro/genact

# another session
$ docker exec -it genact
$ /app/genact
...
// ctrl+c working
```

**Comment by svenstaro** [comment]
This should now be fixed in 0.3.0.
